pkg/signup/signup.go

```diff
@@ -40,6 +40,12 @@ type Signup struct {
 	// Company from the Identity Provider
 	Company string `json:"company"`
 	Status  Status `json:"status,omitempty"`
+	// StartDate is the date that the user's current subscription started, in ISO8601 format
+	StartDate string `json:"startDate,omitempty"`
+	// End Date is the date that the user's current subscription will end, in ISO8601 format
+	EndDate string `json:"endDate,omitempty"`
+	// DaysRemaining is a float representing the number of days remaining in the user's subscription
+	DaysRemaining *float64 `json:"daysRemaining,omitempty"`
```

Comment:
Why do you still keep the pointer for`DaysRemaining` ?

Reply:
A pointer is correct here.  Otherwise how would you differentiate between `DaysRemaining` not being set (i.e. nil) and `DaysRemaining` actually having a value of `0` ?

Reply:
Ok, could you please update the comment so it's clear?

Reply:
updated

Reply:
Why using float64 and not int or uint? Also I wonder if we really need days remaining... EndDate should be enough, right? WDYT?